src/main/webapp/app/exercises/shared/participation/participation.utils.ts

```
@@ -102,7 +103,11 @@
  * @param participation
  * @param showUngradedResults
  */
-export function getLatestResultOfStudentParticipation(participation: StudentParticipation | undefined, showUngradedResults: boolean): Result | undefined {
+export function getLatestResultOfStudentParticipation(
+    participation: StudentParticipation | undefined,
+    showUngradedResults: boolean,
+    showAthenaPreliminaryFeedback: boolean = false,
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)